src/Admin/Settings.php

```diff
@@ -188,7 +191,7 @@ public function plausible_analytics_settings_page() {
 							<label>
 								<?php esc_html_e( 'Shared Link:', 'plausible-analytics' ); ?>
 								<span class="plausible-analytics-admin-field-input">
-									<input style="width: 550px; max-width: 100%;" type="text" name="plausible_analytics_settings[shared_link]" value="<?php echo $shared_link; ?>" />
+									<input style="width: 550px; max-width: 100%;" type="text" name="plausible_analytics_settings[shared_link]" value="<?php esc_attr_e($shared_link); ?>" />
```

Comment:
@Ankur0812 Use `esc_url` instead of `esc_attr_e` as it contains a URL

Reply:
Done